test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -371,6 +371,8 @@ public void test() throws IOException
                                                                         .add(this::selectTokenRange))
                                   .addIf(State::hasEnoughMemtable, StatefulASTBase::flushTable)
                                   .addIf(State::hasEnoughSSTables, StatefulASTBase::compactTable)
+                                  .addAllIf(BaseState::allowRepair, b -> b.add(StatefulASTBase::incrementalRepair)
+                                                                          .addIf(BaseState::isConsistent, StatefulASTBase::previewRepair))
```

Comment:
preview repairs fails if it finds a mismatch, so we can only call it safely in the test when we *know* we should be consistent. 

for accord, we flush during the `prepare` phase, which is *before* we do any barriers, so we can't just say "accord makes us consistent", because we might not be (you need to go through accord to be consistent).